Brokerages/Alpaca/AlpacaBrokerage.DataQueueHandler.cs

```diff
@@ -104,7 +104,7 @@ private static bool CanSubscribe(Symbol symbol)
             if (symbol.ID.SecurityType != SecurityType.Equity)
                 return false;
 
-            return symbol.Value.ToLower().IndexOf("universe", StringComparison.Ordinal) == -1;
+            return symbol.Value.ToLowerInvariant().IndexOf("universe", StringComparison.Ordinal) == -1;
```

Comment:
Same comment about removing the `ToLowerInvariant` and directly calling `IndexOf` with ignore-case set to `true`
- Same for InteractiveBrokersBrokerage.cs:2455 https://github.com/QuantConnect/Lean/pull/3442/files#diff-6384ef48924c357fa33193a7b54c1a40R2455

Reply:
Using `StringExtensions.IndexOfInvariant(string substring, bool ignoreCase)`